src/models/guild.ts

```diff
@@ -4,12 +4,15 @@ export interface GuildConfig {
   _id: string,
   prefix: string,
   canvasInstanceID: string,
-  courseChannels: {
-    CategoryID: string,
-    //Record<courseID, channelID), courseID should be number -> parseInt()
-    channels: Record<string, string>
-  }
+  courseChannels: CourseChannels,
   info: Command[],
   commands: Command[],
   notes: Record<string, string[]>
 }
+
+export interface CourseChannels{
+  CategoryID: string,
+  //Record<courseID, channelID), courseID should be number -> parseInt()
```

Comment:
Since we're going to all string for courseIDs you can change it to just //Record<courseID, channelID)